### PR TITLE
Define custom ERLANG_DIALYZER_APPS that includes sasl and eldap

### PR DESCRIPTION
**Pull request by stevendanna**

### CODE REVIEW

**Comment by stevendanna** [comment]
cc @chef/lob

**Comment by marcparadise** [comment]
:+1: please to merge

**Comment by marcparadise** [comment]
Context: I have some further dialyzer cleanups that should go on top of this

File: src/oc_erchef/custom.mk [hunk 1]
```diff
@@ -9,6 +9,23 @@ REL_HOOK = compile bundle
...
+                       kernel \
+                       mnesia \
+                       public_key \
+                       sasl \
+                       ssl \
+                       stdlib \
+                       syntax_tools \
+                       tools \
+                       xmerl
+
```
Comment (marcparadise) [inline src/oc_erchef/custom.mk]: Depending on how it's set in concrete.mk, you might be able to do: 
ERLANG_DIALYZER_APPS += eldap sasl

**Pull request merged**